support/Pipelines/Bbh/__init__.py

```diff
@@ -21,9 +21,9 @@ def list_commands(self, ctx):
 
     def get_command(self, ctx, name):
         if name == "ecc-control":
-            from .EccentricityControl import ecc_control_fit
+            from .EccentricityControl import eccentricity_control
 
-            return ecc_control_fit
+            return eccentricity_control
```

Comment:
This doesn't work because it's not a CLI command. Either put in the CLI command or delete this for now.